tests/Unit/Models/FamilyModelTest.php

```diff
@@ -6,12 +6,128 @@
 use App\Carer;
 use App\Child;
 use App\Centre;
+use App\Registration;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 
 class FamilyModelTest extends TestCase
 {
     use DatabaseMigrations;
 
+    /** @test */
+    public function itCreditsWhenAFamilyIsPregnant()
+    {
+        // Make a pregnant family
+        $family = factory(Family::class)->create();
+
+        $pregnancy = factory(Child::class, 'unbornChild')->make();
+        $family->children()->save($pregnancy);
+
+        // There should be a credit reason of 'FamilyIsPregnant'
+        $credit_reasons = $family->getCreditReasons();
+        $this->assertEquals(1, count($credit_reasons));
+        $this->assertEquals('pregnant', $credit_reasons[0]["reason"]);
+    }
+
+    /** @test */
+    public function itCanHaveRegistrations()
+    {
+        // Create Family
+        $family = factory(Family::class)->create();
+
+        // There should be no registrations, and that's fine.
+        $this->assertEquals(0, $family->registrations()->count());
+
+        // Associate with two registrations
+        $registration1 = new Registration();
```

Comment:
Is there a registration factory?

Reply:
Yes, but it builds families and other things too. I wanted just a registration.
Factories should probably be split out.

Reply:
:+1: